Assets/MRTK/SDK/Experimental/Elastic/Scripts/ElasticSystem.cs

```diff
@@ -0,0 +1,212 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using Microsoft.MixedReality.Toolkit.Utilities;
+using System;
+using System.Collections.Generic;
+using System.Runtime.CompilerServices;
+using UnityEngine;
+
+
+[assembly: InternalsVisibleTo("Microsoft.MixedReality.Toolkit.Tests.PlayModeTests")]
+namespace Microsoft.MixedReality.Toolkit.Experimental.Physics
+{
+
+    /// <summary>
+    /// Properties of the extent in which a damped
+    /// harmonic oscillator is free to move.
+    /// </summary>
+    [Serializable]
+    public struct ElasticExtentProperties<T>
+    {
+        /// <value>
```

Comment:
Our guidelines advise to separate structs into separate files: https://microsoft.github.io/MixedRealityToolkit-Unity/Documentation/Contributing/CodingGuidelines.html#public-classes-structs-and-enums-should-all-go-in-their-own-files

Reply:
Thanks! Moved structs into ElasticTypes.